server/portal/apps/webhooks/views.py

```diff
@@ -27,6 +27,21 @@
 
 terminal_job_states = ["FINISHED", "CANCELLED", "FAILED"]
 
+class JobState(Enum): 
+    PENDING = "processing"
+    PROCESSING_INPUTS = "processing"
+    STAGING_INPUTS = "queueing"
+    STAGING_JOB = "queueing"
+    SUBMITTING_JOB = "queueing"
+    QUEUED = "queueing"
+    RUNNING = "running"
+    ARCHIVING = "finishing"
+    FINISHED = 'finished'
+    STOPPED = 'stopped'
+    FAILED ='failure'
+    BLOCKED = 'blocked'
+    PAUSED = 'paused'
+    CANCELLED = 'cancelled'
```

Comment:
@rstijerina let me know what you think about this. I added this enum here so we can check for duplicates in the backend based on the front-end text value

Reply:
Hmm, would it be simpler to minimize the accepted statuses in `PORTAL_JOB_NOTIFICATION_STATES`?

Reply:
Hmm, yeah that could be a simpler solution. And then I can just check to see if a status exists in that list to determine whether or not to send a notification. Is that what you were thinking?

Reply:
Yeah, exactly. I think that's happening currently in the webhooks view